__test__/connect.spec.mjs

```diff
@@ -99,6 +99,27 @@ test("forward http2", async (t) => {
   t.assert(res.data.includes(expected));
 });
 
+test("forward http2 no cert validation", async (t) => {
+  const httpServer = await makeHttp({useHttp2: true});
```

Comment:
Nice!

Reply:
yeah, wanted to specifically test here where i know there's a working http2 server 😄  thanks again!